data_and_models/pipelines/ner/spacy_transformers/config.cfg

```diff
@@ -0,0 +1,158 @@
+# Initialized with: spacy init config config.cfg -l en -p ner -o accuracy -G.
+
+# Blue Brain Search is a text mining toolbox focused on scientific use cases.
+#
+# Copyright (C) 2020  Blue Brain Project, EPFL.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+[paths]
+train = null
+dev = null
```

Comment:
This is not related to this PR but I wanted to ask you your opinion @pafonta. I saw that for our "main" models in the ner pipelines, we have a common `config.cfg` for all the models. Do you think it is going to stay like this or it could be we will have several `config.cfg` in the future ? I am sorry, I do not understand quite well all those parameters for now.

Reply:
> we have a common `config.cfg` for all the models. Do you think it is going to stay like this

Yes.

> wanted to ask you your opinion

My opinion is that we could integrate `DVC` + `spaCy` in the following way:

1. We use `params.yaml` from `DVC` to overwrite parameters from the default `config.cfg` from `spaCy`.
2. We have a `Python` script which creates a `config.cfg` with the parameters from `params.yaml` overwritten.
3. This step is a DVC stage.
4. We could use `dvc exp run --set-params`, `dvc run --params`, and `dvc params diff` transparently 🚀 

What do you think?

Reply:
I see! I did not think of it that way but it is really interesting! Thanks for the answer!